setuptools/dep_util.py

```diff
@@ -7,7 +7,7 @@ def __getattr__(name):
     if name not in ['newer_pairwise_group']:
```

Comment:
I'm not certain the point of checking in a list rather than `if name != 'newer_pairwise_group'`. Copied boilerplate from when there's more argument?

Reply:
It is due to copied boilerplate (see `_distutils.dep_util`). It felt better to use a consistent syntax than to re-write the logic, especially considering this code path is deprecated.